terror/terror.go

```diff
@@ -328,6 +328,14 @@ func MustNil(err error) {
 	}
 }
 
+// CleanNotNil cleans up and fatals if err is not nil.
+func CleanNotNil(err error, closeFun func()) {
```

Comment:
Why not name it `FatalNotNil`?
And if the `closeFun` is optional, then we can use it in more places.